### DISCUSSION TITLE
元宇宙好像離我們還有點遠?

### DISCUSSION THREAD

**Discussion by ghost** [1101-GS4719-C/Python-Community-GS4719-C#219] [科技新知]
[英特爾：想實現沉浸式元宇宙？運算效率還得提升 1 千倍](https://technews.tw/2021/12/22/immersive-metaverse-computing-efficiency/)

**Comment by Jen-Siang** [comment]
隨著科技快速發展，相信很快就能看到相關產品陸續推出的

**Comment by Tomlord1122** [comment]
根據摩爾定律，硬體的技術大概一年半到兩年就會增加一倍，所以應該還需要幾年吧

**Reply by wax75397** [reply]
我認為可能會遇到物理上的瓶頸而使技術成長放緩，可能要更多年後才能實現了

**Comment by EricAgumonJou** [comment]
元宇宙需要網路速度、運算速度、5G、AR、VR、區塊鏈以及人工智慧每一項都成熟，才能算真正完成發展～

**Reply by EricAgumonJou** [reply]
這些技術的發展，也都是元宇宙成熟重要的指標。
目前，元宇宙的發展還算從基礎開始而已，要真正實現元宇宙可能還需要段時間。
但是科技的進步，相信讓你我都相當期待。：）

**Comment by luo910227** [comment]
我覺得志祺七七解釋得挺清楚的，我的想法是最近許多科技公司都已經投入了大量研究，再結合5G、AR、VR、MR、XR等技術，雖然元宇宙不是近在咫尺但是應該不遠。


https://www.youtube.com/watch?v=4ScJsxuFuaQ
